src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIPostDraftApprenticeshipOverlapOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using NUnit.Framework;
+using SFA.DAS.ProviderCommitments.Web.Models.OveralppingTrainingDate;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Web.UnitTests.Controllers.OverlappingTrainingDateRequestControllerTests
+{
+    [TestFixture]
+    public class WhenIPostDraftApprenticeshipOverlapOptions
+    {
+        private OverlappingTrainingDateRequestControllerTestFixture _fixture;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _fixture = new OverlappingTrainingDateRequestControllerTestFixture();
+        }
+
+        [Test]
+        public async Task AndWhenWhenUserSelectsToSendOverlapEmailToEmployer()
+        {
+            await _fixture.SetupStartDraftOverlapOptions(OverlapOptions.SendStopRequest).DraftApprenticeshipOverlapOptions();
+            _fixture.VerifyOverlappingTrainingDateRequestEmailSent();
+            _fixture.VerifyUserRedirectedTo("EmployerNotified");
+        }
+
+        [Test]
+        public async Task AndWhenWhenUserSelectsToContactTheEmployer()
+        {
+            await _fixture.SetupStartDraftOverlapOptions(OverlapOptions.ContactTheEmployer).DraftApprenticeshipOverlapOptions();
+            _fixture.VerifyOverlappingTrainingDateRequestEmail_IsNotSent();
+            _fixture.VerifyUserRedirectedTo("Details");
+        }
```

Comment:
Missing test for when the user selects to "Add the apprentice later"

Reply:
done